framework/src/auxkernels/FunctorElementalAux.C

```diff
@@ -28,18 +28,24 @@ FunctorElementalAuxTempl<is_ad>::validParams()
   InputParameters params = AuxKernel::validParams();
   params.addClassDescription(
       "Evaluates a functor (variable, function or functor material property) on the current "
-      "element. For finite volume, this evaluates the material property at the centroid.");
+      "element or quadrature point.");
   params.addRequiredParam<MooseFunctorName>("functor", "The functor to evaluate");
   params.addParam<MooseFunctorName>("factor", 1, "A factor to apply on the functor");
-
+  params.addParam<bool>(
```

Comment:
the default should probably depend on the type of variable this function is used to fill

Reply:
That's an excellent point. Perhaps we shouldn't even have the parameter, and just test casting of the variable in the constructor?